doc/boot-partition.md

```diff
@@ -250,7 +250,29 @@ AI Ihno:
 
 - provide a script to detect DASD type (FBA, DIAG...), is it allowed to create a partition table?
 
+## aarch64 (RPi 3)
 
+- see also fate #323484, bsc #1041475
+
+- RPi boots from sdcard, must have a msdos partition table
```

Comment:
I have looked for the [upstream docs](https://github.com/raspberrypi/documentation/tree/master/hardware/raspberrypi/bootmodes), found it, and it [says:](https://github.com/raspberrypi/documentation/blob/master/hardware/raspberrypi/bootmodes/bootflow.md)

> The boot ROM also now supports GUID partitioning and has been tested with hard drives partitioned using Mac, Windows, and Linux.